sqlstore/src/main/resources/db/migration/V0030__project_shortName.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE Project ADD COLUMN shortName varchar(20) DEFAULT NULL;
```

Comment:
Fixed. The string handling in MySQL makes this impractical. I'll do this in Java.